src/docs/presentations.md

```diff
@@ -1,5 +1,11 @@
 # Presentations
 
+## A Simple Standard for Ontological Mappings 2024: The Case for prioritising 5-Star mappings (Every Cure, internal seminar) 
```

Comment:
```suggestion
## A Simple Standard for Ontological Mappings 2024: The case for prioritising 5-Star mappings (Every Cure, internal seminar) 
```
My personal preference is to have "Case" be lower-case, however, feel free to ignore this suggestion since it does match the title of the slides as-is.